active_learning_single_annotator/active_learning_single_annotator.ipynb

```diff
@@ -0,0 +1,639 @@
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Epoch 0:  87%|████████████████████████████████████████████████████████████████████▏         | 271/310 [00:32<00:04,  8.25it/s, loss=5.49, v_num=]\u001b[A\n",
+      "                                                                                                                                                 \u001b[A"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Epoch 0, global step 10: 'val_accuracy' reached 0.01699 (best 0.01699), saving model to '/home/ubuntu/examples/active_learning_single_annotator/AutogluonModels/ag-20230317_201329/epoch=0-step=10.ckpt' as top 3\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Epoch 0:  87%|████████████████████████████████████████████████████████████████████▏         | 271/310 [00:48<00:07,  5.55it/s, loss=5.49, v_num=]\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "INFO:automm:Start to fuse 2 checkpoints via the greedy soup algorithm.\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Predicting DataLoader 0: 100%|███████████████████████████████████████████████████████████████████████████████████| 13/13 [00:02<00:00,  5.89it/s]\n",
+      "Predicting DataLoader 0: 100%|███████████████████████████████████████████████████████████████████████████████████| 13/13 [00:02<00:00,  5.80it/s]\n",
+      "Predicting DataLoader 0: 100%|███████████████████████████████████████████████████████████████████████████████████| 13/13 [00:02<00:00,  5.71it/s]\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "INFO:automm:Models and intermediate outputs are saved to /home/ubuntu/examples/active_learning_single_annotator/AutogluonModels/ag-20230317_201329 \n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Predicting DataLoader 0: 100%|████████████████████████████████████████████████████████████████████████████████▋| 595/597 [01:40<00:00,  5.90it/s]"
+     ]
+    }
+   ],
+   "source": [
+    "predictor, pred_probs_unlabeled = predict_autogluon_classification( df_labeled,\n",
+    "                                                                    out_folder=None,\n",
+    "                                                                    df_predict=df_unlabeled,\n",
+    "                                                                    time_limit=30)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Obtain active learning scores\n",
+    "\n",
+    "Next, we will get the active learning scores for each datapoint. Since we only have one annotation per example, we only care about the active learning scores for the unlabeled data, the `active_learning_scores_unlabeled`. \n",
```

Comment:
```suggestion
    "Next, we will compute active learning scores that estimate the informativeness of labeling each datapoint. Since we will collect at most one annotation per example, we only care about active learning scores for the unlabeled data: `active_learning_scores_unlabeled`. \n",
```